index.js

```diff
@@ -91,6 +91,7 @@ class SSHConfig extends Array {
   append(opts) {
     let config = this
     let configWas = this
+    const lastParam = opts[Object.keys(opts)[Object.keys(opts).length - 1]]
```

Comment:
`opts[Object.keys(opts).pop()]` would be better. But I believe we don't need to do lastParam check here.

Reply:
To add an extra line break at the bottom of the appended config, we can just do a `config[config.length - 1].after += '\n'` at the rear of this function.